OracleWebLogic/samples/12212-domain/container-scripts/domain.properties

```diff
@@ -0,0 +1,10 @@
+DOMAIN_NAME=abc_domain
+ADMIN_PORT=7001
+ADMIN_HOST=wlsadmin
+CLUSTER_NAME=DockerCluster
+PRODUCTION_MODE=dev
+ADMIN_PASSWORD=welcome4
```

Comment:
Yes the domain.properties is the same as using the -e property in the command line.  I can delete the admin password from domain.properties and have the user enter the admin password using -e.  The pain is that both the Admin Server and Managed Server need to have access to the same password.  If the user is over writing the password they would need to use the -e on both docker run command lines to set the admin password.  If the password is auto generated in the admin server docker run then the user needs to find the password in the docker logs and then pass it with -e command line when running the managed server container.  I am investigating if there is a way around this.